### DISCUSSION TITLE
Orario Docenti

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#574] [Idee e proposte]
Sarebbe possibile inserire la possibilità di aggiungere l'orario dei docenti all'interno del registro?

**Comment by trinko** [comment]
L'orario ho intenzione di inserirlo quest'anno, se tutto fila liscio...

**Reply by Davide031** [reply]
Circa entro quando?

**Reply by trinko** [reply]
Mi spiace, non sono in grado di fare previsioni.

**Reply by sempliciniluigi89** [reply]
Buonasera ,sarebbe possibile provvedere all'integrazione dell'orario dei docenti all'interno del registro?Lo chiedo perché più si va avanti e più io ,come altri docenti ,ci rendiamo conto che ,dover ogni volta consultare il formato cartaceo che tante volte si perde o comunque non si ha a portata di mano ,diventa difficile ,soprattutto per i docenti con più cattedre stabilire l'ora in cui firmare .
